adapters/YahooAdapter/README.md

```diff
@@ -1,8 +1,8 @@
-# Verizon Media Mediation Adapter for Google Mobile Ads SDK for iOS
+# Yahoo Mediation Adapter for Google Mobile Ads SDK for iOS
 
 This is an adapter to be used in conjunction with the Google Mobile Ads SDK.
 For requirements, instructions, and other info, see the
-[Verizon Media Adapter Integration Guide](https://developers.google.com/admob/ios/mediation/verizon-media).
+[Yahoo Adapter Integration Guide](https://developers.google.com/admob/ios/mediation/verizon-media).
```

Comment:
Currently this link points to the Verizon SDK Integration Guide, obviously not part of this PR, please rebrand the web integration guide as well.

Reply:
The mediation developer guide will also be updated.